CONTRIBUTING.md

```diff
@@ -121,7 +121,7 @@ When adding new Babel related translation the implementation is a bit more compl
 2. Next we run the following command to let Babel search for keys:
     * ```pybabel extract -F babel.cfg -o messages.pot .```
 3. We now have to add the found keys to all translation files, with the following command:
-    * ```pybabel update --omit-header -i messages.pot -d translations```
```

Comment:
Deze change willen we wel houden denk ik? Misschien ff een issue aanmaken?

Reply:
Ja zal ik doen!